test/unit/features/directives/once.spec.js

```diff
@@ -0,0 +1,110 @@
+import Vue from 'vue'
+
+describe('Directive v-once', () => {
+  it('should not rerender component', done => {
+    const vm = new Vue({
+      el: '#app',
+      template: '<div v-once>{{ a }}</div>',
+      data: { a: 'hello' }
+    })
+    waitForUpdate(() => {
```

Comment:
the initial rendering is sync so you don't have to wait to assert the initial render.